main/acle.md

```diff
@@ -1406,8 +1417,9 @@ this implies:
 
 ### Javascript floating-point conversion
 
-`__ARM_FEATURE_JCVT` is defined to 1 if the FJCVTZS (AArch64) or
-VJCVT (AArch32) instruction and the associated intrinsic is available.
+`__ARM_FEATURE_JCVT` is defined to 1 if the FJCVTZS (AArch64)
+or VJCVT (AArch32) instruction and the [associated
+intrinsic]#floating-point-data-processing-intrinsics is available.
```

Comment:
I dare grammar :)

```suggestion
intrinsic]#floating-point-data-processing-intrinsics are available.
```

Reply:
Actually, this is missing (…) around the link, oops.  The text is “associated intrinsic”, so the singular itself is right.

Will fix.

Reply:
OK, but I thought that the verb referred to both the instruction *and* the associated intrinsic. That's two things to me :)

Reply:
Fair :-)  I'll fix that while I'm here, thanks.